userspace/libscap/linux/scap_ppm_sc.c

```diff
@@ -420,6 +420,8 @@ static const ppm_sc_code *g_events_to_sc_map[] = {
 	[PPME_SYSCALL_FCHOWNAT_X] = (ppm_sc_code[]){PPM_SC_FCHOWNAT, -1},
 	[PPME_SYSCALL_UMOUNT_1_E] = (ppm_sc_code[]){PPM_SC_UMOUNT, -1},
 	[PPME_SYSCALL_UMOUNT_1_X] = (ppm_sc_code[]){PPM_SC_UMOUNT, -1},
+	[PPME_SYSCALL_UMOUNT2_E] = (ppm_sc_code[]){PPM_SC_UMOUNT2, -1},
```

Comment:
We should also update `[PPME_SYSCALL_UMOUNT_E,X]` entries to NULL ;)

Reply:
Given that they are no more sent.

Reply:
Oh i am wrong, it is an old version and should stay there!

Reply:
uhm, I don't think so, `[PPME_SYSCALL_UMOUNT_E,X]` are old versions so it should be correct to associated them to `PPM_SC_UMOUNT2`

Reply:
In the end, I mapped it to `PPM_SC_UMOUNT`, this entry will be never used and in this way the mapping would be more meaningful `umount_event_pair` -> `ppm_sc_umount`